django_project/ford3/templates/campus.html

```diff
@@ -13,72 +13,118 @@
-                    {% endfor %}
-                    </ul>
+                <div class='col-md-6'>
+                    <p><b>Telephone number:</b> {{ campus.telephone }}</p>
+                    <p><b>Email:</b> {{ campus.email}}</p>
+                    <p><b>Max number of Students per year in Campus:</b> {{ campus.max_students_per_year }}</p>
                 </div>
             </div>
 
 
 
             <div class='row'>
-                <div class='col-md-10'>
-                    <h4>2. Qualification Information</h4>
+                <div class='col-md-10 edu-underline'>
+                    <h4 class="mt-3">2. Location</h4>
                 </div>
-                <div class='col-md-2 text-center'>
-                    <a href="{% url 'edit-campus' provider_id=provider.id campus_id=campus.id  %}?step=campus-qualifications" class='edu-button edu-button-orange p-0'>Edit qualifications</a>
+                <div class='col-md-2'>
+                    <a href="{% url 'edit-campus' provider_id=provider.id campus_id=campus.id  %}?step=campus-location" class='edu-button edu-button-orange p-0'>Edit</a>
                 </div>
             </div>
+            <div class='row'>
+                <div class='col-md-6 py-4'>
+                    <p><b>Physical address:</b></p>
+                    <p>{{ campus.physical_address_line_1 }}</p>
+                    <p>{{ campus.physical_address_line_2 }}</p>
+                    <p>{{ campus.physical_address_city }}</p>
+                    <p>{{ campus.physical_address_postal_code }}</p>
+                </div>
+                <div class="col-md-6 py-3">
+                    {% if campus.location %}
+                        {{ campus.location }}
+                    {% else %}
+                        <table class="table table-bordered h-100 w-50">
+                            <tr>
+                                <td class="align-middle text-center text-muted">No location yet</td>
+                            </tr>
+                        </table>
+                    {% endif %}
+                </div>
 
+            </div>
 
+            <div class='row py-4'>
+                <div class='col-md-10 edu-underline'>
+                    <h4 class="mt-3">3. Important Dates</h4>
+                </div>
+                <div class='col-md-2 text-center'>
+                    <a href="{% url 'edit-campus' provider_id=provider.id campus_id=campus.id  %}?step=campus-dates" class='edu-button edu-button-orange p-0'>Edit</a>
+                </div>
+            </div>
+            {% for event in campus.events %}
+            <div class='row py-3'>
+                <div class='col-md-6'>
+                    <b>{{ event.name }}:</b> {{ event.date_start }} - {{ event.date_end }}
+                </div>
+                <div class='col-md-6'>
+                    <b>Links to event:</b> {{ event.http_link }}
```

Comment:
Do not repeat `Links to event`
It is supposed to be a http link so add a `<a></a>`